outputs/constants.go

```diff
@@ -37,4 +37,5 @@ const (
 	Orange    string = "#ff5400"
 
 	Kubeless string = "Kubeless"
+	Openfaas string = "Openfaas"
```

Comment:
I didn't found out where you use this constant.

For `Kubeless` it's used [HERE](https://github.com/falcosecurity/falcosidekick/blob/c3962df9b4e7d9f6943008c12cfe6a9193a548cc/outputs/client.go#L166)

I think the same code block is missing for your `OpenFaas` client, without, the response will not be logged.

Reply:
done, thank you, we missed it.